arch/arm/src/rp23xx/rp23xx_testset.c

```diff
@@ -36,7 +36,13 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#define RP23XX_TESTSET_SPINLOCK     0   /* Spinlock used for test and set */
+/* Errata RP2350-E2 SIO SPINLOCK writes are mirrored at +0x80 offset
+ * Use only safe SPINLOCKS
+ * The following SIO spinlocks can be used normally as they do not alias
+ * with writable registers: 5, 6, 7, 10,11, and 18 through 31.
+ */
+
+#define RP23XX_TESTSET_SPINLOCK     7   /* Spinlock used for test and set */
```

Comment:
where we use this new macro

Reply:
It was there before and used in `up_testset` function, I've changed the number only.